theme.json

```diff
@@ -341,7 +341,8 @@
 			}
 		},
 		"spacing": {
-			"blockGap": "1.5rem"
+			"blockGap": "1.5rem",
+			"padding": "0 var(--wp--custom--spacing--small)"
```

Comment:
This should be an object with left, right, top, bottom and not a string.

Reply:
It should be moved to `settings.layout.padding` as well.

Reply:
Ah, ok that gets things working right! Thanks.